gnirehtet.cmd

```diff
@@ -0,0 +1,149 @@
+@ECHO OFF
+CALL :locate_file ADB "%LOCALAPPDATA%\Android\sdk\platform-tools\adb.exe"
+CALL :locate_file JAVA "%JAVA_HOME%\bin\java.exe"
+CALL :locate_file APK %~dp0gnirehtet.apk %~dp0app\build\outputs\apk\gnirehtet-release.apk %~dp0app\build\outputs\apk\gnirehtet-debug.apk
+CALL :locate_file RELAY %~dp0relay.jar %~dp0relay\build\libs\relay.jar
+
+::main
+IF NOT "%2"=="" SET serial="-s %2"
+CALL :do_%1 %serial% 2>NUL || CALL :do_help 
```

Comment:
Is it guaranteed that there are no other unrelated `do_something` functions defined somewhere?